bolt/include/bolt/Core/BinaryContext.h

```diff
@@ -660,6 +672,9 @@ class BinaryContext {
 
   std::unique_ptr<MCAsmBackend> MAB;
 
+  /// ICF level to use for this binary.
+  ICFLevel ICFLevelVar{ICFLevel::None};
```

Comment:
It's awkward to call a variable with the suffix Var. Can't you just call it CurrICFLevel or something?